src/tutorials/0006/05.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <div>
+        <MultipleChoiceLesson
+        :text="text"
+        :question="question"
+        :choices="choices" />
+    </div>
+</template>
+
+<script>
+import MultipleChoiceLesson from '../../components/MultipleChoiceLesson'
+import text from './05.md'
+
+const question = 'How can we know the base encoding of the hash?'
+
+const choices = [
+  {
+    answer: 'We have to guess it, but it\'s easy because most implementations of IPFS encode CIDs with "base32."',
```

Comment:
```suggestion
    answer: 'We have to guess it, but it\'s easy because most implementations of IPFS encode CIDs with "base32".',
```

Reply:
I believe this is a British v American English issue. If we can get markdown working here so we can use `base32` and put the period after it'll be perfect. :)

Reply:
No way, I did not know this!